src/containers/Form/FormLayout.tsx

```diff
@@ -134,6 +134,7 @@ export const FormLayout: React.SFC<FormLayoutProps> = ({
   const { loading, error } = useQuery(getItemQuery, {
     variables: variables,
     skip: !itemId,
+    fetchPolicy: 'no-cache', // This is required to restore the data after save
```

Comment:
instead of not going with cache, can we update the cache?

Basically, if you include the things you want to update in the update query it automatically updates the cache.